frontend/src/dispatcher/js-messages.ts

```diff
@@ -320,19 +293,11 @@ export class UpdateImageData extends JsMessage {
 
 export class DisplayRemoveEditableTextbox extends JsMessage {}
 
-export class UpdateDocumentLayer extends JsMessage {
+export class UpdateDocumentLayerDetails extends JsMessage {
 	@Type(() => LayerPanelEntry)
 	readonly data!: LayerPanelEntry;
 }
 
-export class UpdateCanvasZoom extends JsMessage {
-	readonly factor!: number;
-}
-
-export class UpdateCanvasRotation extends JsMessage {
-	readonly angle_radians!: number;
-}
-
 export type BlendMode =
 	| "Normal"
 	| "Multiply"
```

Comment:
I don't think we actually need this definition on the frontend. Or in other words, `LayerPanelEntry` doesn't need a lot of its metadata like `blend_mode` and `opacity`. Should we just `#[serde(skip)]` these and remove them from `js-messages.ts`? @0HyperCube

Reply:
Sure